Sources/ExtensionDictionary.swift

```diff
@@ -24,8 +24,50 @@
 //
 
 import Foundation
-
-internal extension Dictionary {
+public extension Dictionary {
```

Comment:
If you mark any of the functions in the extension public, it gives a warning that the extension must be public as well (at least for me).